proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -297,7 +302,7 @@ message CreateConnectorRequest {
     (buf.validate.field).string.max_len = 128,
     (buf.validate.field).string.pattern = "^[a-zA-Z0-9-_]+$",
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      description: "Unique name of target connect cluster. For Redpanda self-hosted deployments, use the cluster name defined in the console's configuration file. For Redpanda Cloud, use `redpanda`.",
+      description: "Unique name of target connect cluster. For Redpanda Cloud, use `redpanda`.",
```

Comment:
For creating a connector we have the following message
```proto
message CreateConnectorRequest {
  string cluster_name = 1 [
    (google.api.field_behavior) = REQUIRED,
    (buf.validate.field).required = true,
    (buf.validate.field).string.min_len = 1,
    (buf.validate.field).string.max_len = 128,
    (buf.validate.field).string.pattern = "^[a-zA-Z0-9-_]+$",
    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
      description: "Unique name of target connect cluster. For Redpanda self-hosted deployments, use the cluster name defined in the console's configuration file. For Redpanda Cloud, use `redpanda`.",
      example: "\"redpanda\"",
      field_configuration: {path_param_name: "cluster_name"}
    }
  ];
  ConnectorSpec connector = 2;
}
```
We need a cluster name and a connector spec.

If we take a look at the spec:

```proto
// ConnectorSpec is the spec of the connector, as defined in the Kafka connect
// API, it can be used as input of the connector creation or output of the
// connectors
message ConnectorSpec {
  string name = 1 [
    (google.api.field_behavior) = REQUIRED,
    (buf.validate.field).required = true
  ];

  map<string, string> config = 2 [
    (google.api.field_behavior) = REQUIRED,
    (buf.validate.field).required = true
  ];
  repeated TaskInfo tasks = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
  string type = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
}
```

We can see that `tasks` and `type` are for "output only" and I believe rapidoc ignores them for like "POST" requests.

Furthermore, if we look at how the create REST API is configured in the proto:

```proto
// CreateConnector implements the create connector method, and exposes an
  // equivalent REST endpoint as the Kafka connect API endpoint
  rpc CreateConnector(CreateConnectorRequest) returns (CreateConnectorResponse) {
    option (google.api.http) = {
      post: "/v1alpha1/connect/clusters/{cluster_name}/connectors"
      body: "connector"
      response_body: "connector"
    };
    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
      summary: "Create connector"
      description: "Create a connector with the specified configuration."
      responses: {
        key: "201"
        value: {
          description: "Created"
          schema: {
            json_schema: {ref: ".redpanda.api.dataplane.v1alpha1.ConnectorSpec"}
          }
        }
      }
    };
  }
```

the `cluster_name` from the `CreateConnectorRequest` message is mapped into the `cluster_name` path param, and then `connector` is mapped into the request JSON body...

So then really the REST api becomes a 

```
POST /v1alpha1/connect/clusters/{cluster_name}/connectors
```

where JSON body is something like:

```json
{
  "config": {},
  "name": "string"
}
```

Hope that explains things?